app/views/reports/new.html.erb

```diff
@@ -0,0 +1,29 @@
+<h2>Add a Report:</h2>
```

Comment:
it would have been better to have it in a modal box.

Reply:
ok sir will work on it.